x-pack/plugins/apm/public/application/action_menu/anomaly_detection_setup_link.tsx

```diff
@@ -58,37 +59,33 @@ export function AnomalyDetectionSetupLink() {
 }
 
 export function MissingJobsAlert({ environment }: { environment?: string }) {
-  const { data = DEFAULT_DATA, status } = useFetcher(
-    (callApmApi) =>
-      callApmApi({
-        endpoint: `GET /api/apm/settings/anomaly-detection/jobs`,
-      }),
-    [],
-    { preservePreviousData: false, showToastOnError: false }
-  );
+  const {
+    anomalyDetectionJobsData = DEFAULT_DATA,
```

Comment:
I was calling my `INITIAL_STATE` instead of `DEFAULT_DATA`, which one you think is better?

Reply:
Either is fine with me. I copied this from what was there before.